.github/workflows/super-mode-calculator.yaml

```diff
@@ -58,9 +58,8 @@ jobs:
 
       - uses: guardian/actions-riff-raff@v2
```

Comment:
worth updating to v4?

Reply:
yes but I think I'll save it for later :)